lib/rules/property-allowed-list/README.md

```diff
@@ -4,39 +4,40 @@ Specify a list of allowed properties.
 
 <!-- prettier-ignore -->
 ```css
-a { display: block; }
+a { color: red; }
 /** ↑
  * This property */
 ```
 
-This rule ignores variables (`$sass`, `@less`, `--custom-property`).
+This rule ignores preprocessor variables (e.g. `$sass`, `@less`).
+It parses custom properties (e.g. `--foo: red`).
```

Comment:
```suggestion
This rule ignores preprocessor variables (e.g. `$sass`, `@less`).
```

Let's keep it consistent with the other `*-property` rules that don't explicitly mention custom properties.